manifests/init.pp

```diff
@@ -94,7 +91,7 @@
   # everytime the settings file changes due to non-report processor reasons (like e.g. if the ServiceNow
   # credentials change). We also return the current report processor checksum so that we can persist it
   # in the settings file.
-  $settings_file_path = "${puppet_base}/servicenow_reporting.yaml"
+  $settings_file_path = "${settings::confdir}/servicenow_reporting.yaml"
```

Comment:
Does this default to `/etc/puppetlabs/puppet/puppet.conf` ?

Reply:
I think this reference should work ok. https://puppet.com/docs/puppet/5.5/configuration.html#confdir

Reply:
Added reference.